reference/tools/tiup/manage-component.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Use TiUP to Manage Components
+summary: Learn how to manage components using TiUP.
+category: tools
+---
+
+# Use TiUP to Manage Components
+
+You can use the following TiUP sub-commands to manage components:
+
+- list: Queries the component list. By using this sub-command, you can see all the optional components to install and all the optional versions of each component.
```

Comment:
```suggestion
- list: Queries the component list. By using this command, you can see all the optional components to install and all the optional versions of each component.
```
I think we can safely use "command" instead of "sub-command" because the meanings are both clear here. But to keep the term usage unified, I suggest using "(TiUP) command". What do you think? @TomShawn

Reply:
Good idea.